internal/monitors/elasticsearch/client/indexstatsfetcher.go

```diff
@@ -0,0 +1,482 @@
+		out = append(out, indexStats.Primaries.Get.getIndexGetStats(enhancedStatsForIndexGroupsOption[GetStatsGroup], defaultDimsForPrimaries)...)
+		out = append(out, indexStats.Primaries.Search.getIndexSearchStats(enhancedStatsForIndexGroupsOption[SearchStatsGroup], defaultDimsForPrimaries)...)
+		out = append(out, indexStats.Primaries.Merges.getIndexMergesStats(enhancedStatsForIndexGroupsOption[MergesStatsGroup], defaultDimsForPrimaries)...)
+		out = append(out, indexStats.Primaries.Refresh.getIndexRefreshStats(enhancedStatsForIndexGroupsOption[RefreshStatsGroup], defaultDimsForPrimaries)...)
+		out = append(out, indexStats.Primaries.Flush.getIndexFlushStats(enhancedStatsForIndexGroupsOption[FlushStatsGroup], defaultDimsForPrimaries)...)
+		out = append(out, indexStats.Primaries.Warmer.getIndexWarmerStats(enhancedStatsForIndexGroupsOption[WarmerStatsGroup], defaultDimsForPrimaries)...)
+		out = append(out, indexStats.Primaries.QueryCache.getIndexQueryCacheStats(enhancedStatsForIndexGroupsOption[QueryCacheStatsGroup], defaultDimsForPrimaries)...)
+		out = append(out, indexStats.Primaries.FilterCache.getIndexFilterCacheStats(enhancedStatsForIndexGroupsOption[FilterCacheStatsGroup], defaultDimsForPrimaries)...)
+		out = append(out, indexStats.Primaries.Fielddata.getIndexFielddataStats(enhancedStatsForIndexGroupsOption[FieldDataStatsGroup], defaultDimsForPrimaries)...)
+		out = append(out, indexStats.Primaries.Completion.getIndexCompletionStats(enhancedStatsForIndexGroupsOption[CompletionStatsGroup], defaultDimsForPrimaries)...)
+		out = append(out, indexStats.Primaries.Segments.getIndexSegmentsStats(enhancedStatsForIndexGroupsOption[SegmentsStatsGroup], defaultDimsForPrimaries)...)
+		out = append(out, indexStats.Primaries.Translog.getIndexTranslogStats(enhancedStatsForIndexGroupsOption[TranslogStatsGroup], defaultDimsForPrimaries)...)
+		out = append(out, indexStats.Primaries.RequestCache.getIndexRequestCacheStats(enhancedStatsForIndexGroupsOption[RequestCacheStatsGroup], defaultDimsForPrimaries)...)
+		out = append(out, indexStats.Primaries.Recovery.getIndexRecoveryStats(enhancedStatsForIndexGroupsOption[RecoveryStatsGroup], defaultDimsForPrimaries)...)
+		out = append(out, indexStats.Primaries.IDCache.getIndexIDCacheStats(enhancedStatsForIndexGroupsOption[IDCacheStatsGroup], defaultDimsForPrimaries)...)
+		out = append(out, indexStats.Primaries.Suggest.getIndexSuggestStats(enhancedStatsForIndexGroupsOption[SuggestStatsGroup], defaultDimsForPrimaries)...)
+		out = append(out, indexStats.Primaries.Percolate.getIndexPercolateStats(enhancedStatsForIndexGroupsOption[PercolateStatsGroup], defaultDimsForPrimaries)...)
+	}
+
+	defaultDimsForTotal := utils.MergeStringMaps(defaultDims, map[string]string{
+		"aggregation": Total,
+	})
+	out = append(out, indexStats.Total.Docs.getIndexDocsStats(enhancedStatsForIndexGroupsOption[DocsStatsGroup], defaultDimsForTotal)...)
```

Comment:
`indexStats.Total` and `indexStats.Primaries` are the same struc type so you can just assign one or the other to a variable and then call `.Docs`, `.Store`, etc. on that with the appropriate dimension set.

Reply:
Assigning to a variable now